lib/hutch/cli.rb

```diff
@@ -65,9 +65,13 @@ def load_app
     def load_rails_app(path)
       # path should point to the app's top level directory
       if File.directory?(path)
-        # Smells like a Rails app if it's got a config/environment.rb file
+        # Smells like a Rails app if it's got a script/rails or bin/rails file
+        is_rails_app = false
+        ['script/rails', 'bin/rails'].each do |file|
+          is_rails_app = true if File.exist?(File.expand_path(File.join(path, file)))
+        end
```

Comment:
Perhaps this can golfed to an `any?` call? Something like

```ruby
is_rails_app = ['script/rails', 'bin/rails'].any? do |file|
  File.exist?(File.expand_path(File.join(path, file)))
end
```